lib/atm.ex

```diff
@@ -18,7 +18,24 @@ defmodule ATM do
   Taken from codewars
   """
   def withdraw(n) do
-    # TODO : Code your solution
-    [0, 0, 0]
+    bills([0, 0, 0], n)
+  end
+
+  defguard test?(number, divisor) when number >= divisor and rem(number, divisor) == 0
```

Comment:
@netikular 🥇for you defguard!

Reply:
it was so hard to give it a name!! 

I finally went with a generic "test?" what would you call this guard?

Reply:
hmm it's hard!  well having the context, will something like:

defguard changeable?(amount, bill) when amount >= bill and rem(amount, bill) == 0
defguard dispensible?(amount, bill) when amount >= bill and rem(amount, bill) == 0

Reply:
or likely_changeble? lol